test/metabase/models/field_values_test.clj

```diff
@@ -22,6 +22,59 @@
    [toucan2.tools.with-temp :as t2.with-temp])
   (:import (clojure.lang ExceptionInfo)))
 
+(def ^:private base-types-without-field-values
```

Comment:
Listing all these here with the following test forces devs adding new types that currently meet the conditions for exclusion to check that the types should actually be excluded.